config/post_updates/10005/paragraphs.paragraphs_type.oe_banner.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
+status: true
+dependencies: {  }
+id: oe_banner
+label: Banner
+icon_uuid: null
+icon_default: null
+description: 'Allows editors to create banner blocks composed of: title, description, call to action and optional image.'
```

Comment:
I think we should describe what the banner is rather than what it has, something like: "Allows editors to create Banners that display a prominent message and related action." (Taken from the ECL description)